openupgrade_scripts/scripts/account/15.0.1.2/post-migration.py

```diff
@@ -52,6 +52,40 @@ def set_res_company_account_setup_taxes_state_done(env):
         company.account_setup_taxes_state = "done"
 
 
+def _handle_website_legal_page(env):
+    """As the terms and conditions page has been added in the 'account' module,
+    the information added in the page created by 'website_legal_page' has to be
+    transferred to the invoice_terms_html field of the company.
+    """
+    openupgrade.update_module_names(
+        env.cr, [("website_legal_page", "account")], merge_modules=True
+    )
+    for company in env["res.company"].with_context(active_test=False).search([]):
+        openupgrade.logged_query(
+            env.cr,
+            """
+            SELECT iuv.id
+            FROM ir_ui_view iuv
+                JOIN website w ON w.id = iuv.website_id
+            WHERE iuv.key = 'website_legal_page.legal_page' AND w.company_id = %s
+            """,
+            (company.id,),
+        )
+        row = env.cr.fetchone()
+        if row:
+            view = env["ir.ui.view"].browse(row[0])
+            view_temp = view.copy(
+                {"arch_db": view.arch.replace('t-call="website.layout"', "")}
```

Comment:
Add a comment on the trick we're doing here so we can insert it into the field without issues.